server/src/main/java/com/lunchtime/models/Restaurant.java

```diff
@@ -1,6 +1,7 @@
 package com.lunchtime.models;
 
 import java.time.Instant;
+import java.util.Set;
```

Comment:
You can use Analyze -> Inspect code in IDEA to find all unused imports and remove your redundant code

Reply:
Made it.